app/server.go

```diff
@@ -2060,12 +2060,12 @@ func withMut(mut *sync.Mutex, f func()) {
 	f()
 }
 
-func cancelTask(mut *sync.Mutex, task *model.ScheduledTask) {
+func cancelTask(mut *sync.Mutex, taskPointer **model.ScheduledTask) {
 	mut.Lock()
 	defer mut.Unlock()
-	if task != nil {
-		task.Cancel()
-		task = nil
+	if *taskPointer != nil {
+		(**taskPointer).Cancel()
```

Comment:
Why `**` here? This should work with a pointer method as well right?

Reply:
Yep, a single `*` works as well.